client/src/Components/EmployeeProfile/EmployeeProfile.scss

```diff
@@ -0,0 +1,103 @@
+@import '../../scss/config';
+
+
+
+
+//mobile first
+.employee-profile{
+    width: 100%;
+    height: auto;
+}
+header {
+        display: flex;
+        margin-bottom: 2rem;
+        padding: 2rem;
+        .logo__main {
```

Comment:
We need to make our css style consistent. Please check the button component and try to align your css writing style with it. You can also check the better way of using modifiers there.